sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -327,20 +327,31 @@ function toPublicPolicy(policy: CoreCertificatePolicy = {}): CertificatePolicy {
 }
 
 function toPublicIssuer(issuer: IssuerBundle = {}): CertificateIssuer {
+  const parsedId = parseKeyvaultEntityIdentifier("certificates", issuer.id);
+  const attributes: IssuerAttributes = issuer.attributes || {};
+
   const publicIssuer: CertificateIssuer = {
     id: issuer.id,
+    name: parsedId.name,
     provider: issuer.provider,
```

Comment:
Only on properties (won't repeat, though maybe this is the last instance).

Reply:
Should be good now :)